yb-voyager/src/srcdb/pg_dump_extract_schema.go

```diff
@@ -135,7 +135,7 @@ func parseSchemaFile(exportDir string, schemaDir string, exportObjectTypesList [
 				objSqlStmts["SEQUENCE"].WriteString(stmts)
 			case "INDEX", "INDEX ATTACH":
 				objSqlStmts["INDEX"].WriteString(stmts)
-			case "TABLE", "DEFAULT", "CONSTRAINT", "FK CONSTRAINT":
+			case "TABLE", "DEFAULT", "CONSTRAINT", "FK CONSTRAINT", "CHECK CONSTRAINT":
```

Comment:
Is this observed for some specific case now?  or in-general for Check constraints in the schema.

Reply:
Yeah observed in case of `rna` database;.
Previously i think it came because of CONSTRAINT type, but CHECK CONSTRAINT is also there.